lib/screens/surveydetails/widget/answer_intro_widget.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:flutter/material.dart';
+
+class AnswerIntroWidget extends StatefulWidget {
```

Comment:
What do you think about using `StatelessWidget` here? 🤔

Reply:
Updated in [b127948](https://github.com/nimblehq/flutter-ic-lydia-ryan/pull/88/commits/b12794892c45840ff3c81c6bc1c979eb0853c0f8) 🙏🏻